src/lib/route.js

```diff
@@ -135,11 +158,21 @@ class Route {
       points: this.getPoints(),
       min_message_window: this.minMessageWindow,
       source_account: this.sourceAccount,
+
       destination_precision: this.destinationPrecision,
-      destination_scale: this.destinationScale
+      destination_scale: this.destinationScale,
+
+      added_during_epoch: this.addedDuringEpoch
```

Comment:
If this field is needed, be sure to add it to https://github.com/interledgerjs/five-bells-shared/blob/master/schemas/Routes.json.